storage-blobs-python-quickstart-v4/README.md

```diff
@@ -0,0 +1,53 @@
+
+# Contributing
+
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a
+Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
+a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
+provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+# How to run this project
+
+## Prerequisites
+
+To complete this quickstart: 
+
+* Install [Azure Storage SDK for Python](https://github.com/Azure/azure-storage-python).
```

Comment:
https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-blob

This is the new link for storage blobs.